sos/report/plugins/__init__.py

```diff
@@ -1569,6 +1569,21 @@ def add_blockdev_cmd(self, cmds, devices='block', timeout=300,
                              whitelist=whitelist, blacklist=blacklist,
                              tags=_dev_tags)
 
+    def add_tape_cmd(self, cmds, devices='tape', timeout=300,
+                     sizelimit=None, chroot=True, runat=None, env=None,
+                     binary=False, prepend_path=None, whitelist=[],
+                     blacklist=[], tags=[]):
```

Comment:
Can't be `add_blockdev_cmd` enhanced accordingly? As we basically duplicate its code here.

Reply:
That was my thought too, we can add `tape` as a known keyword for `devices`. Otherwise, this looks good and `get_tape_devs()` is a nice addition.